cli.js

```diff
@@ -71,6 +71,7 @@ var cli = meow([
 	'  --watch, -w      Re-run tests when tests and source files change',
 	'  --source, -S     Pattern to match source files so tests can be re-run (Can be repeated)',
 	'  --timeout, -T    Set global timeout',
+	'  --pool-size, -z  Set worker pool size (BETA)',
```

Comment:
I personally don't remember us getting any feedback on the watch trial. It was pretty much only us on the team using it. I think we should document it as `(EXPERIMENTAL)`. That way we're free to do whatever we want with it.